portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Projects</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="content">
+      <h1>Projects</h1>
+      <div id="proj-container">
+        <div class="default-slide">
+          <div class="default-slide-block">
+            Click the forward and backward arrows to view my projects.
+          </div>
+        </div>
+        <div class="slide">
```

Comment:
This looks like a similar possible opportunity to merge this and its nested div together